README.md

```diff
@@ -46,30 +46,31 @@ all of which are compatible with the Apache 2.0 Licence.
-| librsvg       | LGPLv3                                                                                                   |
-| libtiff       | [libtiff License](http://www.libtiff.org/misc.html) (BSD-like)                                           |
-| libuuid       | New BSD License                                                                                          |
-| libvips       | LGPLv3                                                                                                   |
-| libwebp       | New BSD License                                                                                          |
-| libxml2       | MIT Licence                                                                                              |
-| pango         | LGPLv3                                                                                                   |
-| pixman        | MIT Licence                                                                                              |
-| zlib          | [zlib Licence](https://github.com/madler/zlib/blob/master/zlib.h)                                        |
+| Library       | Used under the terms of                                                                                   |
+|---------------|-----------------------------------------------------------------------------------------------------------|
+| cairo         | Mozilla Public License 2.0                                                                                |
+| expat         | MIT Licence                                                                                               |
+| fontconfig    | [fontconfig Licence](https://gitlab.freedesktop.org/fontconfig/fontconfig/blob/master/COPYING) (BSD-like) |
+| freetype      | [freetype Licence](https://git.savannah.gnu.org/cgit/freetype/freetype2.git/tree/docs/FTL.TXT) (BSD-like) |
+| fribidi       | LGPLv3                                                                                                    |
+| gdk-pixbuf    | LGPLv3                                                                                                    |
+| gettext       | LGPLv3                                                                                                    |
+| giflib        | MIT Licence                                                                                               |
+| glib          | LGPLv3                                                                                                    |
+| harfbuzz      | MIT Licence                                                                                               |
+| lcms          | MIT Licence                                                                                               |
+| libcroco      | LGPLv3 (Windows only)                                                                                     |
```

Comment:
libcroco also still seems to be used on macOS, see:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/librsvg.rb#L19

Shall I change it into?:
```suggestion
| libcroco      | LGPLv3 (Windows / macOS only)                                                                                     |
```

Reply:
Yes please.

Reply:
Resolved with: https://github.com/lovell/sharp-libvips/pull/33/commits/1a37a8b7b43b0ddfb12e2c8a0ff6cb13bef291e7.